web3/contracts.go

```diff
@@ -145,9 +162,42 @@ func (c *Contracts) LoadContracts(addresses *Addresses) error {
 	if err != nil {
 		return fmt.Errorf("failed to bind process registry: %w", err)
 	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), web3QueryTimeout)
+	defer cancel()
+
+	// check vkey used by sequencer is the same as the one used by the contract
+	vkey, err := process.GetVerifierVKeyHash(&bind.CallOpts{Context: ctx})
```

Comment:
No 'Get'

Reply:
Yes 'Get' because it comes directly from the solidity bindings where the convention is to use Get<x>

Reply:
I can create a wrapper if you want, so the naming will be more consistent with Go style, lmk

Reply:
ah ok, understood